packages/nextjs/app/page.tsx

```diff
@@ -32,6 +85,44 @@ const Home: NextPage = () => {
             <span className="font-bold">Checked in builders count:</span>
             <span>{checkedInCounterElement()}</span>
           </p>
+          <div className="flex flex-col text-center text-md">
+            <div>
+              {!isConnectedOptimism ? (
+                <div className="mt-4 text-xl font-bold"> Please Connect to Optimism To Read List of Builders </div>
+              ) : (
+                <>
+                  <span className="font-extrabold text-lg">List of Builers</span>
+                  {isReadingEventLoading ? (
```

Comment:
We could add this check to remove flickering of UI : 

`{isReadingEventLoading || !eventHistory?.length ? (`

Reply:
Haha it shows up underneath the text for me.. are you viewing from a smaller screen? I will have a look anything and see if the issue is responsivness

Reply:
Hey @technophile-04  - i get what you mean. I am trying to fix this but I am unable to commit my code. I get the following error:
hooks/scaffold-eth/useScaffoldEventSubscriber.ts(33,5): error TS2322: Type 'Abi' is not assignable to type 'InferContractAbi<Contract<TContractName>>'.
I don't think I touched that hook so not sure what the issue is. Also, I have just committed not 10/15 mins ago without issue (although I had to pull before I could commit) . Any ideas?